src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -156,22 +193,30 @@ module.exports = class LevelPlane extends Array {
 
     // If in bounds, we want to see if any redstone is around the index in question
     // Below index
-    if (orthogonalBlocks.south.block !== undefined && orthogonalBlocks.south.block.blockType.startsWith("redstoneWire")) {
+    if (orthogonalBlocks.south.block !== undefined &&
+    (orthogonalBlocks.south.block.isRedstone ||
+    orthogonalBlocks.south.block.blockType === 'railsRedstoneTorch')) {
```

Comment:
Would be good to encode as much of this into the `LevelBlock` model as we can.  Maybe we want `affectsRedstone` / `affectedByRedstone` flags?

I don't want this conditional to get even more gnarly when we add pressure plates 😄.

Reply:
I was thinking about having a helper function that we could add conditionals to, and just call within those if statements? Rather than go and add different flags to a bunch of different blocks?

Either solution would be fine with me.

Reply:
Think of `LevelBlock` as an extensible type.  Right now we do `new LevelBlock("redstoneWire")` but there's no reason we couldn't change that to `new RedstoneWireBlock()` (where `RedstoneWireBlock extends LevelBlock`).

Following this pattern, we'll want to use polymorphism to track how each type behaves, vs. adding a helper function in `LevelPlane`.  Then we can ask a block about its behavior anywhere, without requiring the helper function.